services/src/main/java/org/keycloak/models/workflow/DeleteUserStepProvider.java

```diff
@@ -17,20 +17,27 @@
 
 package org.keycloak.models.workflow;
 
+
 import org.jboss.logging.Logger;
 import org.keycloak.component.ComponentModel;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.RealmModel;
 import org.keycloak.models.UserModel;
-
+import org.keycloak.models.UserProvider;
+import org.keycloak.storage.DatastoreProvider;
+import org.keycloak.storage.datastore.DefaultDatastoreProvider;
 
 public class DeleteUserStepProvider implements WorkflowStepProvider {
 
+    public static final String PROPAGATE_TO_SP = "PROPAGATE_TO_SP";
```

Comment:
I would suggest using lowe case with dashes for these config constants - we've been changing a bit the workflow definition to use lower case constants everywhere as it reads better in the final YAML or JSON definition.

So we are, for example, moving away from `USER_LOGGED_IN` in favor of `user-logged-in`. In this case, I would set the constant to something like `propagate-to-storage-provider` or just `propagate-to-provider`.